src/v0/sources/shopify/util.js

```diff
@@ -145,6 +144,38 @@ const getAnonymousIdFromDb = async (message, metricMetadata) => {
   }
   return getAnonymousId(message);
 };
+/**
+ * This function sets the sessionId based on cart_token or id from the properties of message.
+ * @param {*} message
+ * @returns
+ */
+const getSessionIdFromDB = async (message, metricMetadata) => {
+  const cartToken = getCartToken(message);
+  const { event } = message;
+  if (!isDefinedAndNotNull(cartToken)) {
+    return null;
+  }
+  let sessionId;
+  try {
+    sessionId = await RedisDB.getVal(`${cartToken}`, 'sessionId');
+  } catch (e) {
```

Comment:
Let's also log the error, to make sure we understand the problem ? WDYT ?! If it occupies too much of logs, we can put it as a debug log and switch the debug logging when needed

Reply:
There can be many logs if redis throws error in cases when `redis is down` which will occupy a lot of logs. `Debug` logs makes sense

Reply:
Done